kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/TritonServerServiceOptions.java

```diff
@@ -99,16 +99,7 @@ public TritonServerServiceOptions(final Map<String, Object> properties) {
     }
 
     public String getAddress() {
-        String address = "";
-        if (this.isLocal) {
-            address = "localhost";
-        } else {
-            final Object propertyAddress = this.properties.get(PROPERTY_ADDRESS);
-            if (propertyAddress instanceof String) {
-                address = ((String) propertyAddress).trim();
-            }
-        }
-        return address;
+        return getStringProperty(PROPERTY_ADDRESS);
```

Comment:
Do we need a trim?

Reply:
Not necessarily but I don't think it hurts. Do you think it can have some negative side effects?

Reply:
Sorry, I was answering under the assumption that `getStringProperty` had the `trim`inside. Fixing